EventFilter/SiPixelRawToDigi/python/SiPixelRawToDigi_cfi.py

```diff
@@ -2,21 +2,6 @@
 import EventFilter.SiPixelRawToDigi.siPixelRawToDigi_cfi
 
 siPixelDigis = EventFilter.SiPixelRawToDigi.siPixelRawToDigi_cfi.siPixelRawToDigi.clone()
-siPixelDigis.Timing = cms.untracked.bool(False)
-siPixelDigis.IncludeErrors = cms.bool(True)
-siPixelDigis.InputLabel = cms.InputTag("siPixelRawData")
-siPixelDigis.UseQualityInfo = cms.bool(False)
-## ErrorList: list of error codes used by tracking to invalidate modules
-siPixelDigis.ErrorList = cms.vint32(29)
-## UserErrorList: list of error codes used by Pixel experts for investigation
-siPixelDigis.UserErrorList = cms.vint32(40)
-##  Use pilot blades
-siPixelDigis.UsePilotBlade = cms.bool(False)
-##  Use phase1
-siPixelDigis.UsePhase1 = cms.bool(False)
-## Empty Regions PSet means complete unpacking
-siPixelDigis.Regions = cms.PSet( ) 
```

Comment:
While all other parameters are set identical, in the fillDescriptions() this is not an empty PSet